rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Every `Rental` object is an intermediary class to the `Book` and `Person` classes. Remember that every `Book` object and `Person` object created has an array named `@rentals`. This array is used to keep track of all the rentals made by that Person object or all the times a Book object was rented. This is the reason why the Book and Person classes have a many-to-many relationship through the Rental class. To keep track of the rentals made by a person, at initialization, you need to push the Rental object itself into the Book and Person rentals array. Kindly implement this:

#### Hint

```ruby
def initialize(date, book, person)
  @date = date
  @book = book
  @person = person
  @book.rentals << self
  @person.rentals << self
end
```